### DISCUSSION TITLE
warpinator enhancement

### DISCUSSION THREAD

**Discussion by Patpoch** [linuxmint/discussions#90] [Development]
Bonjour,
I installed warpinator and tested with several computers , when downloading files with the same name from from different computers , the last one wins and the previous ones are just erased by the last one.
I would like to propose a new possibility , making warpinator create a sub-directory (if it does not exist) for each machine and under that , a directory for each different transfer date, at the end you would have :
$HOME/warpinator/machine1/transfer date/remote content
That could be 2 new options and would be very useful . I'am thinking about saving cell phone's data or getting files with the same name from different computers.
I made it work (but I did not add the options, just hard coded the stuff) , it was not that difficult.
Thanks for your comments
Patrick

**Comment by mtwebster** [comment]
I hadn't considered receiving files with the same name from different sources, this is something I'd be ok with adding as an option. For most users this would probably be overkill, but we should also at least rename files that end up duplicates during a race, or when the overwrite protection option is disabled.
